samples/gateway/Gateway_6/Headquarters/Program.cs

```diff
@@ -0,0 +1,52 @@
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.DependencyInjection;
+using NServiceBus.Gateway;
+using Shared;
+
+Console.Title = "Headquarters";
+var builder = Host.CreateApplicationBuilder(args);
+
+var endpointConfiguration = new EndpointConfiguration("Samples.Gateway.Headquarters");
+endpointConfiguration.EnableInstallers();
+endpointConfiguration.UseSerialization<XmlSerializer>();
+endpointConfiguration.UseTransport(new LearningTransport());
+
+#region HeadquartersGatewayConfig
+var gatewayConfig = endpointConfiguration.Gateway(new NonDurableDeduplicationConfiguration());
+gatewayConfig.AddReceiveChannel("http://localhost:25899/Headquarters/");
+gatewayConfig.AddSite("RemoteSite", "http://localhost:25900/RemoteSite/");
```

Comment:
gatewayConfig.AddSite("RemoteSite", "http://localhost:25900/RemoteSite/");
Any particular reason why you are using a different port?